### PR TITLE
Add uni-directional volume-coupled CFD->transport testcase

### PR DESCRIPTION
**Pull request by uekerman**
As promised, a uni-directional volume-coupled CFD->transport testcase. To have a somehow interesting volume coupling testcase and in the mid-future, we could do dynamic meshes in the transport solver.
Joint work with @gertjanvanzwieten 

@gertjanvanzwieten Do you still want to tweak Nutils stuff?
@MakisH What did we still forget?

Who wants to add an OpenFOAM CFD case? :grin:

### CODE REVIEW

**Comment by fsimonis** [comment]
@IshaanDesai and I will work on a adaptive version in a separate branch, so this can be merged from our side.

**Review by IshaanDesai — APPROVED** [review]
I ran this case and everything seems to be in order :+1: I am working on the remeshing variant of the transport participant in the branch [remeshing_volume_nutils_case](https://github.com/precice/tutorials/tree/remeshing_volume_nutils_case) and will open another PR for it, so we can merge this one :+1:

**Comment by uekerman** [comment]
@MakisH still wanted to have a loook. Please merge if you agree.

**Review by MakisH — COMMENTED** [review]
That's a very nice addition! Very simple, yet very good example of volume coupling, which will be quite important later on.

I looked over the Nutils scripts and they look reasonable, to the extent I understand Nutils. I also left a few minor comments here and there, with the most important to decide now being the name of the directory.

I ran the case and it seems to be working fine. I then used `/usr/bin/time` to measure the runtime and memory consumption, since this seems to be quite a heavy case (great workbench for some performance optimizations). In the end, I got (for `max-time=0.1`):

```
# fluid-nutils
259.72user 69.47system 4:20.69elapsed 126%CPU (0avgtext+0avgdata 327216maxresident)k
648inputs+26496outputs (10major+60725minor)pagefaults 0swaps

# transport-nutils
201.40user 56.23system 3:51.01elapsed 111%CPU (0avgtext+0avgdata 127080maxresident)k
0inputs+46584outputs (3major+27997minor)pagefaults 0swaps
```
meaning that the simulation till 0.1s took a bit more than 4min, and needed 454MB (327+127) of memory.

**Comment by MakisH** [comment]
Also: don't forget to make this tutorial available via the sidebar of the website.

File: channel-transport/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Channel transport
```
[the remark this answers is not in the document]
Comment (uekerman) [reply channel-transport/README.md:2]: Discussed with @MakisH, here for reference: We want to let the name reflect the physics of the case as much as possible. In this sense, `channel-transport` is good though not perfect. `volume-coupled-diffusion` is maybe suboptimal in that sense, but this is also due to the fact that the physics of that are also completely artificial.
Adding `volume` to the name might give problems in the long run for:

- scenarios with both volume and surface coupling (something, we want strategically),
- geometric multi-scale cases, where the coupling can be both a volume and a surface coupling depending on the point of view. 

We decided to keep `channel-transport` for the moment, but renaming some tutorial cases might be necessary in the long run, especially if we collect more and more (which is good).

File: tools/check-size.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ for tutorial in $tutorials; do
   for img in $images; do
     actualsize=$(du -k "$img" | cut -f 1)
     # Check gifs
-    if [[ "${img}" == *.gif ]]; then
+    if [[ "${img}" == *.gif || "${img}" == *.webp ]]; then
```
[the remark this answers is not in the document]
Comment (uekerman) [reply tools/check-size.sh:23]: This one @fsimonis added.

**Review by MakisH — APPROVED** [review]

File: tools/check-size.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (fsimonis) [reply tools/check-size.sh:23]: I already compressed the animation quite a lot but wasn't aware of webm. Will have a look at it

[the remark this answers is not in the document]
Comment (fsimonis) [reply tools/check-size.sh]: I converted the animation to webm and I am not sure if it displays correctly in the GitHub markdown. There is support for an embedded video tag, so we will see.

**Pull request merged**